### PR TITLE
Use JobAccessorV2 in JobConfigActions

### PR DESCRIPTION
**Pull request by gkillough**

### CODE REVIEW

File: src/test/java/com/synopsys/integration/alert/web/api/job/JobConfigActionsTest.java [hunk 1]
```diff
@@ -516,30 +525,11 @@ public void validateJobsByIdEmptyListTest() {
...
-
-        Mockito.doThrow(new AlertException("Exception for Alert test")).when(fieldModelProcessor).performAfterReadAction(Mockito.any());
-
-        ActionResponse<List<JobFieldStatuses>> actionResponse = jobConfigActions.validateJobsById(jobIdsRequestModel);
-
-        assertTrue(actionResponse.isError());
-        assertEquals(HttpStatus.INTERNAL_SERVER_ERROR, actionResponse.getHttpStatus());
-        assertFalse(actionResponse.hasContent());
-    }
-
```
Comment (JakeMathews) [inline src/test/java/com/synopsys/integration/alert/web/api/job/JobConfigActionsTest.java:537]: Why is this no longer needed?

**Review by JakeMathews — APPROVED** [review]

File: src/test/java/com/synopsys/integration/alert/web/api/job/JobConfigActionsTest.java (hunk 1 above)
Comment (gkillough) [reply src/test/java/com/synopsys/integration/alert/web/api/job/JobConfigActionsTest.java:537]: The method `JobConfigActions.convertJobDatabaseModelToRestModel(...)`, the method that threw the exception this test was for, is no longer used in `JobConfigActions.validateJobsById(...)`. So there is no longer a need to test exception handling as there is none.

File: alert-database/src/main/java/com/synopsys/integration/alert/database/api/StaticJobAccessorV2.java [hunk 2]
```diff
@@ -248,6 +248,7 @@ private DistributionJobModel createJobWithId(UUID jobId, DistributionJobRequestM
     }
 
     private DistributionJobModel convertToDistributionJobModel(DistributionJobEntity jobEntity) {
+        UUID jobId = jobEntity.getJobId();
```
Comment (bamandel) [inline alert-database/src/main/java/com/synopsys/integration/alert/database/api/StaticJobAccessorV2.java:251]: It might be better to have this listed closer to where it's actually used

File: web/src/main/java/com/synopsys/integration/alert/web/api/job/JobFieldModelPopulationUtils.java [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+                                 .map(UUID::toString)
+                                 .orElse(null);
+        return new JobFieldModel(jobIdString, Set.of(providerFieldModel, channelFieldModel));
+    }
+
+    public static void populateProviderFields(FieldModel providerFieldModel, DistributionJobModel jobModel) {
+        String providerCommonConfigId = Optional.ofNullable(jobModel.getBlackDuckGlobalConfigId())
+                                            .map(String::valueOf)
+                                            .orElse(null);
+        putField(providerFieldModel, "provider.common.config.id", providerCommonConfigId);
```
Comment (bamandel) [inline web/src/main/java/com/synopsys/integration/alert/web/api/job/JobFieldModelPopulationUtils.java:65]: Are all of these constants no longer available in this package? Using the same constants would cut down on any potential misspellings.

File: web/src/main/java/com/synopsys/integration/alert/web/api/job/JobFieldModelPopulationUtils.java [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+import com.synopsys.integration.alert.common.persistence.model.job.details.JiraCloudJobDetailsModel;
+import com.synopsys.integration.alert.common.persistence.model.job.details.JiraServerJobDetailsModel;
+import com.synopsys.integration.alert.common.persistence.model.job.details.MSTeamsJobDetailsModel;
+import com.synopsys.integration.alert.common.persistence.model.job.details.SlackJobDetailsModel;
+import com.synopsys.integration.alert.common.rest.model.FieldModel;
+import com.synopsys.integration.alert.common.rest.model.FieldValueModel;
+import com.synopsys.integration.alert.common.rest.model.JobFieldModel;
+import com.synopsys.integration.alert.descriptor.api.BlackDuckProviderKey;
+
+public class JobFieldModelPopulationUtils {
```
Comment (bamandel) [inline web/src/main/java/com/synopsys/integration/alert/web/api/job/JobFieldModelPopulationUtils.java:47]: Do we want to make util classes final? I don't see any reason why we'd want someone extending this.

**Review by bamandel — APPROVED** [review]

File: web/src/main/java/com/synopsys/integration/alert/web/api/job/JobFieldModelPopulationUtils.java (hunk 3 above)
Comment (gkillough) [reply web/src/main/java/com/synopsys/integration/alert/web/api/job/JobFieldModelPopulationUtils.java:65]: Ugh lol. I stole a lot of this from the DB util which didn't have access to these field constants. I'll fix it.
